doc/docs/master/deploy-manage/manage/expose-pach-ui-ingress.md

```diff
@@ -0,0 +1,71 @@
+# Expose the Pachyderm UI Through an Ingress Resource
+
+Before completing the steps in this section, read [Overview](../configure-external-access/).
+
+A Kubernetes Ingress enables you to allow external traffic inside
+of a Pachyderm cluster. Specifically, if you do not want to use
+port-forwarding, you might want to expose the Pachyderm UI, `dash`
+through an Ingress resource. To do so, you need to deploy an ingress
+controller and ingress resource object on the Kubernetes cluster that
+runs your Pachyderm cluster. While Kubernetes supports multiple ingress
+controllers, not all of them might work seamlessly with Pachyderm.
+See [Pachyderm Ingress Requirements](../configure-external-access/#pachyderm-ingress-requirements/) for more information.
+
+In addition, follow these recommendations:
+
+* `DaemonSets` are generally more preferred than `Deployments`.
+* You can deploy the `Ingress` resource in any namespace.
+* You need to have administrative access to the hostname that you
+specify in the `hostname` field in the `Ingress` resource `.yaml`.
+* Do not create routes (`paths`) with `pachd` services or S3 services
+in the `Ingress` resource `.yaml`.
+
+This section provides an example of how to deploy a traefik ingress
+controller in minikube.
+
+!!! note
+    If you cannot use traefik, you can try to deploy another ingress
+    controller, such as NGINX, using the provided example.
+
+To expose the Pachyderm UI through a traefik resource, complete
+the following steps:
+
+1. Deploy a Pachyderm cluster to a cloud provider of your choice as
+described in [Deploy Pachyderm](../../deploy/).
+For production deployments, deploy with the `--tls` flag.
+1. Enable authentication as described in [Configure Access Controls](../../../enterprise/auth/auth/).
+1. Deploy the traefik ingress controller:
+
+   ```bash
+   kubectl apply -f https://raw.githubusercontent.com/pachyderm/pachyderm/master/examples/traefik-ingress/roles.yaml
+   kubectl apply -f https://raw.githubusercontent.com/pachyderm/pachyderm/master/examples/traefik-ingress/traefik-daemonset.yaml
+   ```
+
+1. Deploy the Ingress resource. The following text is an example
+metadata file. You need to adjust it to your deployment:
+
+   <script src="https://gist.github.com/svekars/9c665571b665bbd999e050cb829adf2e.js"></script>
```

Comment:
I can't see this gist. even when I view file on github.  is it only viewable on our docs site?  Be nice to solve that.

I saved the gist embed to a local file and viewed it, and it looks fine and correct!